tests/dialects/test_hive.py

```diff
@@ -328,7 +328,6 @@ def test_time(self):
                 "presto": "DATE_ADD('DAY', 1, CAST(CAST('2020-01-01' AS TIMESTAMP) AS DATE))",
                 "hive": "DATE_ADD('2020-01-01', 1)",
                 "spark": "DATE_ADD('2020-01-01', 1)",
-                "": "TS_OR_DS_ADD('2020-01-01', 1, 'DAY')",
```

Comment:
leave this test and see what it transpiles to?

Reply:
Sure, I think the only difference is that the `DAY` unit is not generated as a string but as a var.